csharp/private/macros/setup_basic_nuget_package.bzl

```
@@ -0,0 +1,94 @@
+load("//csharp/private:providers.bzl", "CSharpAssembly")
+load("//csharp/private:rules/imports.bzl", "import_library", "import_multiframework_library")
+
+def _import_dll(dll, has_pdb, imports):
+    path = dll.split("/")
+
+    tfm = path[1]
+
+    # Ignore frameworks we don't support (like net35)
+    if tfm not in CSharpAssembly:
+        return
+
+    lib_name = path[-1].rsplit(".", 1)[0]
+
+    target_name = "%s-%s" % (lib_name, tfm)
+
+    if lib_name not in imports:
+        imports[lib_name] = {tfm: target_name}
+    else:
+        imports[lib_name][tfm] = target_name
+
+    if dll in has_pdb:
+        import_library(
+            name = target_name,
+            target_framework = tfm,
+            dll = dll,
+            pdb = dll[:-3] + "pdb",
+        )
+    else:
+        import_library(
+            name = "%s-%s" % (lib_name, tfm),
```

Is this name intended to be different, or can it also be set to `target_name`?

```suggestion
            name = target_name,
```

---

Thanks, left-over from refactoring.